lang/syn/src/idl/file.rs

```diff
@@ -159,7 +159,9 @@ pub fn parse(
                 .map(|arg| {
                     let mut tts = proc_macro2::TokenStream::new();
```

Comment:
you can adjust the `to_idl_type` function so it accepts a `syn::Type` instead of a `syn::Field` and then you can use it here

Reply:
👍